### PR TITLE
Additional mongodb admin tests

### PR DESCRIPTION
**Pull request by PaulineMauryL**

### CODE REVIEW

File: server/lomas_server/tests/test_mongodb_admin.py [hunk 1]
```diff
@@ -526,6 +646,77 @@ def test_del_dataset(self) -> None:
...
+
+        add_dataset(
+            self.db,
+            dataset,
+            database_type,
+            metadata_database_type,
+            dataset_path=dataset_path,
+            metadata_path=metadata_path,
+        )
+        dataset_found = show_dataset(self.db, "PENGUIN")
```
Comment (damienbfs) [inline server/lomas_server/tests/test_mongodb_admin.py:668]: Should we rename the "show" functions to "get"?

**Review by damienbfs — APPROVED** [review]
Nice! (Merci d'être resté dans le thème de Tintin et Milou^^)

File: server/lomas_server/tests/test_mongodb_admin.py (hunk 1 above)
Comment (PaulineMauryL) [reply server/lomas_server/tests/test_mongodb_admin.py:668]: yes, good idea

Comment (PaulineMauryL) [reply server/lomas_server/tests/test_mongodb_admin.py:668]: i started but it is such a mess with the notebooks, documentation and all.. I will juste create a ticket and there will a PR just for that to keep it simpler

**Pull request merged**